src/clr-angular/forms/combobox/options.ts

```diff
@@ -41,12 +41,13 @@ export class ClrOptions extends AbstractPopover implements OnDestroy {
   }
 
   private initializeSubscriptions(): void {
-    this.sub = this.ifOpenService.ignoredElementChange.subscribe((el: ElementRef) => {
-      if (el) {
-        this.ignoredElement = el.nativeElement;
-      }
-      this.sub.unsubscribe();
-    });
+    // todo: fix on combobox merge
```

Comment:
nit: related to the feedback above about why not adding the ignoredElement now.

Reply:
Same as above - I prefer to fix it in real context, with the real services, real tests. This is undead code atm :)

Reply:
Updated to Jeremy's proposed comment style